src/Servers/Kestrel/Core/src/Internal/Http3/Http3ControlStream.cs

```diff
@@ -245,6 +254,48 @@ private async ValueTask<long> TryReadStreamHeaderAsync()
         }
     }
 
+    private async ValueTask<long> TryReadWebTransportStreamSessionIdAsync()
```

Comment:
It's a Function with Identical Implementation in Both Control and Request Streams; struggling to find an ideal location